.github/renovate.json5

```diff
@@ -0,0 +1,82 @@
+{
+  $schema: 'https://docs.renovatebot.com/renovate-schema.json',
+
+  baseBranches: ['next'],
+  enabledManagers: ['npm'],
+  extends: ['config:base', 'group:allNonMajor'],
+  fetchReleaseNotes: false, // Only fetch release notes for breaking changes
+  internalChecksFilter: 'strict', // PRs will be skipped unless a non-pending version is available (applies to "stabilityDays" check only)
+  labels: ['npm-dependencies'],
+  prConcurrentLimit: 1,
+  repositories: ['chanzuckerberg/edu-design-system'],
+  rangeStrategy: 'bump', // Always bump minor/patch to latest
+  reviewers: ['team:edu-design-system'],
```

Comment:
If this syntax doesn't work to assign a group, then one option could be to make edu-design-system codeowners of the package.json file and then they'll get auto assigned.

Reply:
Oooh thats a nice workaround, will keep in mind, thanks